projects/driver_controls/src/can_fsm.c

```diff
@@ -17,10 +17,10 @@ FSM_STATE_TRANSITION(state_can_transmit) {
 }
 
 static void prv_transmit_data(FSM *fsm, const Event *e, void *context) {
-  InputEventData *data = (InputEventData *)&e->data;
-
-  printf("Device = %d, State = %d, Data = %d\n", e->id, data->components.state,
-         data->components.data);
+  printf("Device = %d, State = %d, Data = %d\n",
+          e->id - 18,
```

Comment:
The CAN event IDs were defined in the same enum as the input events, so the values needed to be unique. However, I think it might be better to split them into their own enums at this point.